test/lib/code_corps/messages/messages_test.exs

```diff
@@ -180,23 +180,23 @@ defmodule CodeCorps.MessagesTest do
 
       result_ids =
         Conversation
-        |> Messages.list_conversations(%{"status" => "active"})
+        |> Messages.list_conversations(%{"active" => true})
         |> get_and_sort_ids()
 
       refute conversation_started_by_admin_without_reply.id in result_ids
       assert conversation_started_by_admin_with_reply.id in result_ids
       assert conversation_started_by_user_without_reply.id in result_ids
       assert conversation_started_by_user_with_reply.id in result_ids
 
-      result_ids =
-        Conversation
-        |> Messages.list_conversations(%{"status" => "any"})
-        |> get_and_sort_ids()
+      # result_ids =
```

Comment:
Should the default be `Repo.all()`....do we want to get all conversations?

Reply:
I think the test on L128 still stands: `returns all records by default`.

In this case we just want to test filtering by that status.

Reply:
And to expand on this, the client really should be responsible for passing the "defaults" in cases where it's not about authorization to see records.